### ISSUE TITLE
Sentiment RNN - description error

### ISSUE THREAD

**Issue by gittar** [udacity/dlnd-issue-reports#238]
In the notebook for Projekt 3 it says:
With train, validation, and text fractions of 0.8, 0.1, 0.1, the final shapes should look like:

                    Feature Shapes:
Train set:          (20000, 200) 
Validation set:     (2500, 200) 
Test set:           (2501, 200)

However, after removal of the (1) zero-length description and its label the test set in may case is also 2500:

		Feature Shapes:
Train set: 		(20000, 200) 
Validation set: 	(2500, 200) 
Test set: 		(2500, 200)

**Comment by gittar** [comment]
concerns not the notebook for project 3, but the notebook for sentiment RNN

**Comment by mcleonard** [comment]
Fixed this!
